README.adoc

```diff
@@ -0,0 +1,37 @@
+= Cray Site Initializer (`csi`)
+:toc:
```

Comment:
I think the adoc may not jive with the auto-deployed documentation.  It can probably be adjusted for it, or we could switch it to mkdocs and there's likely a plugin that works with it.

The docs are built automatically every merge, but it is mostly for command code changes. (https://cray-hpe.github.io/cray-site-init/)

I think  we can up our docs game to match vshasta2

Reply:
Good call. I didn't look at that failure yet - was just working on basecamp. I'll check out adoc support or revert this.

Reply:
It doesn't like the go:build stanzas after a block comment.

Reply:
I assume you were talking about the github-pages build failure.